test/Freezr.spec.js

```diff
@@ -0,0 +1,30 @@
+import {
+  buildMockObjectFrozen,
+  buildMockObjectDeepFrozen,
+  buildMockArrayFrozen,
+  // buildMockArrayDeepFrozen,
+  expectToBeImmutableAndThrowError
+} from './helpers'
+
+describe('freezr', function () {
+  it('returns an immutable object', () => {
+    const { frozen } = buildMockObjectFrozen()
```

Comment:
the point of the test is to freeze an Object and check if it's frozen so it'll be better to see the source object to improve readability.